### DISCUSSION TITLE
How to use slot rendering?

### DISCUSSION THREAD

**Discussion by narceclm** [leezng/vue-json-pretty#194] [Q&A] [answered]
Hi, i want to use a custom slot for rendering. Is there any demo o tutorial for that? How that works?

Thanks a lot!

**Comment by leezng** [comment, marked as answer]
If using vue3, you can see this. https://github.com/leezng/vue-json-pretty/blob/dev/example/Basic.vue

**Reply by wq3911** [reply]
if using vue2 ,how can I use it?

**Reply by leezng** [reply]
https://github.com/leezng/vue-json-pretty/blob/928e39d522e0bb8688be7ea4d2fb69cdd1a7c034/example/Basic.vue#L73

**Reply by wq3911** [reply]
thanks

**Reply by wq3911** [reply]
It's not work with Vue2

**Reply by liyajing228** [reply]
> https://github.com/leezng/vue-json-pretty/blob/928e39d522e0bb8688be7ea4d2fb69cdd1a7c034/example/Basic.vue#L73

it worked
